english/experience/black-point/black-point.md

```diff
@@ -2,8 +2,8 @@ You gaze at the black point for a while.
 
 However, nothing happen, it's just a little black point.
 
-What do you do now?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.